apprentice_teacher.rb

```diff
@@ -1,23 +1,22 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'refactored_code'
+#Getting error: /Users/apprentice/Desktop/phase-1-checkpoint-challenge-3-teacher-refactoring/apprentice_teacher.rb:1:in
+#`require_relative': /Users/apprentice/Desktop/phase-1-checkpoint-challenge-3-teacher-refactoring/refactored_code.rb:38:
+#syntax error, unexpected keyword_end (SyntaxError)
+
+class ApprenticeTeacher < Person
```

Comment:
Looks like apprentice & teacher share lots of attributes & a few methods. How can you refactor this even more?